python/fledge/services/core/api/plugins/update.py

```diff
@@ -65,7 +65,7 @@ async def update_plugin(request: web.Request) -> web.Response:
         # if status is -1 (Already in progress) then return as rejected request
         result_payload = {}
         action = 'update'
-        package_name = "fledge-{}-{}".format(_type, name.lower())
+        package_name = "fledge-{}-{}".format(_type, name.lower().replace('_', '-'))
```

Comment:
not sure, if we should replace _ with - where we expect package name only (not the plugin name); may be better to reject if such plugin is not installed and let client fix it.